perl/bin/cgpVaf.pl

```diff
@@ -171,8 +162,7 @@ sub option_builder {
                 'chr|chromosome=s{,}' => \@{$options{'chr'}},
                 'ct|concat=i'  => \$options{'ct'},
                 'o|outDir=s'  => \$options{'o'},
-                'm|augment=i' => \$options{'m'},
-                'ao|augment_only=i' => \$options{'ao'},
```

Comment:
What happens if this argument is still used, warnings errors etc, change to command line is not allowed in a hotfix.